.github/workflows/preview.yml

```diff
@@ -80,8 +90,98 @@ jobs:
           name: dist
           path: ./dist/
           if-no-files-found: error
-          retention-days: 7
 
       - name: Publish to TestPyPI
         if: inputs.TestPyPI
         run: poetry publish --repository testpypi --username __token__ --password ${{ secrets.TESTPYPI_API_TOKEN }}
+
+  build_windows:
+    name: Build Windows standalone binary
+    if: inputs.CompileWindows
+    needs: publish_preview
+    runs-on: windows-latest
+    steps:
+      - name: Checkout the repo
+        uses: actions/checkout@692973e3d937129bcbf40652eb9f2f61becf3332 # v4.1.7
+
+        # Nuitka needs the packaged form and not the editable install Poetry provides
+        # Ref: https://github.com/Nuitka/Nuitka/issues/2965
+      - name: Download build artifacts
+        uses: actions/download-artifact@fa0a91b85d4f404e444e00e005971372dc801d16 # v4.1.8
+
+      - name: Set up Python
+        uses: actions/setup-python@f677139bbe7f9c59b41e40162b753c062f5d49a3 # v5.2.0
+        with:
+          python-version: ${{ env.PYTHON_VERSION }}
+
+      - name: Install poetry
+        run: pipx install --python ${{ env.PYTHON_VERSION }} poetry==${{ env.POETRY_VERSION }}
+
+      - name: Configure poetry
+        run: poetry config virtualenvs.in-project true
+
+      - name: Install the project with poetry
+        run: |
+          poetry check --lock
+          poetry lock --no-update --no-cache
+          poetry install --verbose --no-root --sync --with compile
+          poetry run python -m pip install --find-links dist --no-index phylum
+
+      - name: Compile binary with Nuitka
+        env:
+          PREVIEW_VER: ${{ needs.publish_preview.outputs.next_ver }}
+        run: |
+          poetry run python -m nuitka `
```

Comment:
Seems like nuitka is preinstalled and already used for the other systems?

Reply:
`nuitka` is a Python package and was added to this repository as an additional dependency in `pyproject.toml`, as part of the new `compile` group. It is only available when poetry installs the package with the `compile` group. Then, running it as `poetry run python -m nuitka` ensures that the "correct Python" ([see their docs for reference](https://nuitka.net/user-documentation/user-manual.html#recommended-way)) is the one that runs it.